figures/4.figure4/scripts/figure4.r

```diff
@@ -0,0 +1,903 @@
+suppressPackageStartupMessages(suppressWarnings(library(ggplot2)))
+suppressPackageStartupMessages(suppressWarnings(library(dplyr)))
+suppressPackageStartupMessages(suppressWarnings(library(patchwork)))
+suppressPackageStartupMessages(suppressWarnings(library(cowplot)))
+suppressPackageStartupMessages(suppressWarnings(library(RcppTOML)))
+suppressPackageStartupMessages(suppressWarnings(library(pheatmap)))
+suppressPackageStartupMessages(suppressWarnings(library(lattice)))
+suppressPackageStartupMessages(suppressWarnings(library(RColorBrewer)))
+suppressPackageStartupMessages(suppressWarnings(library(gplots)))
+suppressPackageStartupMessages(suppressWarnings(library(ComplexHeatmap)))
+suppressPackageStartupMessages(suppressWarnings(library(ggplotify)))
+suppressPackageStartupMessages(suppressWarnings(library(viridis)))
+suppressPackageStartupMessages(suppressWarnings(library(platetools)))
+suppressPackageStartupMessages(suppressWarnings(library(circlize)))
+suppressPackageStartupMessages(suppressWarnings(library(reshape2)))
+suppressPackageStartupMessages(suppressWarnings(library(stringr)))
+suppressPackageStartupMessages(suppressWarnings(library(purrr)))
```

Comment:
Consider adding a comment to describe why these suppressions are necessary. Alternatively, is there a configuration which might avoid them having to be listed here directly (and instead could they be applied across R files somehow)?